app/app/context.py

```diff
@@ -61,7 +64,10 @@ def preprocess(request):
         if record_visit:
             ip_address = get_ip(request)
             profile.last_visit = timezone.now()
-            profile.save()
+            try:
```

Comment:
What are we guarding against here with the try block?

Reply:
theres been some intermittent issue with a non-serializable object coming out of `profille.to_dict`- i can't reliably reproduce it.. i think its a caching issue; but i threw this try/except in in case it pops up in prod -- i could get a profile of it from sentry